test/e2e/base_user_test.go

```diff
@@ -19,19 +19,18 @@ import (
 
 type baseUserIntegrationTest struct {
 	suite.Suite
-	hostAwait    *wait.HostAwaitility
-	memberAwait  *wait.MemberAwaitility
-	member2Await *wait.MemberAwaitility
+	awaitilities Awaitilities
```

Comment:
I believe the code could be even simpler if we used something like this:
```
type baseUserIntegrationTest struct {
	suite.Suite
	Awaitilities
}
```
and then instead of `hostAwait := s.awaitilities.Host()` we would have `hostAwait := s.Host()`.

And one step further, if we rename the new `Awaitilities.Host()` method to `Awaitilities.HostAwait()`, then the old `s.hostAwait` becomes `s.HostAwait()` and we could get rid of the extra `hostAwait := s.awaitilities.Host()` statements and simply do 

```
s.HostAwait().CreateWithCleanup(context.TODO(), bannedUser)
```

for example

Reply:
It's much better indeed, thanks for the suggestion! 👍  Done in c7fd9ef45d20ff26b25476830325e59722253eac